### ISSUE TITLE
Quantization

### ISSUE THREAD

**Issue by fakerybakery** [xai-org/grok#15]
Hi,
Thanks for releasing Grok! Is there any chance we could load the model in 4-bit given how large it is? Do you know if bitsandbytes support is planned (cc @timdettmers)?
Thanks!

**Comment by deadpipe** [comment]
Yes, is there a quantization support to run on consumer GPU (eg: 3090, 4090)?

**Comment by StavrosD** [comment]
The model needs about 300GB ram using 8 bit quantization. 
4-bit quantization will half the size to 150GB. You have to use multiple GPUs , at least 7 consumer GPUs with 24GB VRAM each.
If we are lucky there will be a gguf model avaiable soon so we will be able to load the model on the computer RAM and offload some layers on the VRAM.

**Comment by AayushSameerShah** [comment]
And I am hoping it to run on my RTX 4050 laptop GOU 😬🤦‍♂️
